backend/src/main/java/com/nikogrid/backend/controllers/AuthenticationController.java

```diff
@@ -0,0 +1,40 @@
+package com.nikogrid.backend.controllers;
+
+import com.nikogrid.backend.dto.AuthenticationDTO;
+import com.nikogrid.backend.dto.RecoveryJwtTokenDTO;
+import com.nikogrid.backend.dto.RegisterDTO;
+import com.nikogrid.backend.exceptions.DuplicateUserException;
+import com.nikogrid.backend.services.AuthenticationService;
+import jakarta.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/v1/auth")
+@Validated
+public class AuthenticationController {
+    private AuthenticationService authenticationService;
```

Comment:
```suggestion
    private final AuthenticationService authenticationService;
    
```

Mark the dependency as `final` and add a newline to separate from the constructor.